config/locales/en.yml

```diff
@@ -748,14 +748,14 @@ en:
           fire_station: "Fire Station"
           food_court: "Food Court"
           fountain: "Fountain"
-          fuel: "Filling Station"
-          gambling: "Gambling"
-          grave_yard: "Grave Yard"
+          fuel: "Gas Station"
```

Comment:
Some time ago, I was showing someone OSM; they were unfamiliar with this archaic term and misread it as “filing station”. It was during tax season, so they thought it the feature was miscategorized as a government office. More recently, I was chatting with someone who snarked that it has something to do with a cafeteria.